LiveRide crashes the app (after upgrading AndroidX/Jetpack)

'Nuff said...

I'm looking at this one now

Found the issue:
```
     Caused by: android.view.InflateException: Binary XML file line #2 in net.cyclestreets:layout/compassbutton: Binary XML file line #2 in net.cyclestreets:layout/compassbutton: Error inflating class android.support.design.widget.CoordinatorLayout
     Caused by: android.view.InflateException: Binary XML file line #2 in net.cyclestreets:layout/compassbutton: Error inflating class android.support.design.widget.CoordinatorLayout
     Caused by: java.lang.ClassNotFoundException: android.support.design.widget.CoordinatorLayout
```

Probably this happened because the initial migration Jez did was before we added this button.

Fix coming right up.  Also (excellently) it turns out that setting `viewBinding.enabled = true` makes this a compile-time failure 🎉  so will protect us against similar errors in future.

--- a/build.gradle
+++ b/build.gradle
@@ -1,5 +1,5 @@
 buildscript {
-  ext.kotlinVersion = '1.3.61'
+  ext.kotlinVersion = '1.3.72'
   repositories {
     google()
     jcenter()

--- a/build.gradle
+++ b/build.gradle
@@ -94,5 +94,7 @@ subprojects {
       // Kotlin (nullability) interop check - potentially useful but quite noisy
       // check 'Interoperability'
     }
+
+    viewBinding.enabled = true
   }
 }

--- a/libraries/cyclestreets-view/src/main/res/layout/compassbutton.xml
+++ b/libraries/cyclestreets-view/src/main/res/layout/compassbutton.xml
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.design.widget.CoordinatorLayout
+<androidx.coordinatorlayout.widget.CoordinatorLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <android.support.design.widget.FloatingActionButton
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
         android:id="@+id/compass_button"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
@@ -18,4 +18,4 @@
         app:ico_color="?android:textColorSecondary"
         app:backgroundTint="@android:color/background_light" />
 
-</android.support.design.widget.CoordinatorLayout>
+</androidx.coordinatorlayout.widget.CoordinatorLayout>

